src/app/scripts/cac/user/cac-user-preferences.js

```diff
@@ -1,67 +1,75 @@
-CAC.User.Preferences = (function(Storages, _) {
+CAC.User.Preferences = (function(_) {
     'use strict';
 
-    // TODO: figure out local storage strategy
-    // set up local storage
-    // var namespaceStorage = Storages.initNamespaceStorage('cac_otp');
-    // var storage = namespaceStorage.localStorage;
 
     // Initialize preference storage object.
-    // Currently it just uses an 'options' dictionary, so preferences lives only as long as
-    // the page for which this is initialized.
+    // All values in storages should be stringified first.
+    // Setting to a falsy value will remove the object from storages;
```

Comment:
All values passed in to this are stringified first (see comment; happens in public preference setter), so an empty string and undefined are the only possible falsy input, and an empty string always means "not set."

Zero can be stored fine because we're actually storing `'0'`.

We should keep the stringifying, if for no other reason, as it will let us switch back to using browser storage if we want.